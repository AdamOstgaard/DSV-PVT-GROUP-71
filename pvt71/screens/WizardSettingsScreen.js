import React from "react";
import AppSingleButton from "../components/AppSingleButton";
import { StyleSheet, Text, View } from "react-native";

export default class WizardVerifyContactScreen extends React.Component {
  static navigationOptions = {
    header: null,
  };
  
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.heading}>Wizard Settings Screen.</Text>
        <View style={styles.bottom}>
          <AppSingleButton
            title="Next"
            onPress={() =>
              this.props.navigation.navigate("Home")
            }
          />
        </View>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#fff"
  },
  heading: {
    fontSize: 35,
    flex: 1,
    justifyContent: "flex-start",
    left: 10,
    marginTop: 50
  },
  bottom: {
    flex: 1,
    justifyContent: "flex-end"
  }
});
