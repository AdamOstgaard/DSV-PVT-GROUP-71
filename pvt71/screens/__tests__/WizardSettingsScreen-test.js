import "react-native";
import React from "react";
import  WizardSettingsScreen  from "../WizardSettingsScreen";
import renderer from "react-test-renderer";

describe("Wizard Setting Screen tests", () => {
    describe("Snapshot test", () => {
        it("renders correctly", () => {
        const tree = renderer.create(<WizardSettingsScreen />).toJSON();
        expect(tree).toMatchSnapshot();
        });
    });
});