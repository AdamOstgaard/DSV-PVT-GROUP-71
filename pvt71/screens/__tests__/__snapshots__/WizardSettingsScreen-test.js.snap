// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wizard Setting Screen tests Snapshot test renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "flex": 1,
        "fontSize": 35,
        "justifyContent": "flex-start",
        "left": 10,
        "marginTop": 50,
      }
    }
  >
    Wizard Settings Screen.
  </Text>
  <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "flex-end",
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#0d6f73",
          "height": 60,
          "opacity": 1,
          "textAlign": "right",
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 30,
            "paddingRight": 10,
            "paddingTop": 8,
          }
        }
      >
        Next
      </Text>
    </View>
  </View>
</View>
`;
